rosbag2_transport/include/rosbag2_transport/recorder.hpp

```diff
@@ -186,6 +189,8 @@ class Recorder : public rclcpp::Node
   bool event_publisher_thread_should_wake();
 };
 
+std::string type_hash_to_string(const rosidl_type_hash_t & type_hash);
```

Comment:
I think make sense to add `ROSBAG2_TRANSPORT_EXPORT` or `ROSBAG2_TRANSPORT_PUBLIC` macro for `type_hash_to_string()` if it is going to be available externally.